app/controllers/dashboards_controller.rb

```diff
@@ -5,6 +5,8 @@ class DashboardsController < ApplicationController
 
   before_action :pick_line_if_not_set, only: [:index, :search]
 
+  SEARCH_LIMIT = 15
```

Comment:
hmm, I see where your head's at, but I think I'd do this slightly differently - the LIMIT 15 is almost a safety feature (makes it harder to get mass records out) and it would be good to preserve as the default behavior. So to that end, what if:
* we open up patient_searchable and change the search method to take an argument, `limit`, with a default value of `SEARCH_LIMIT`
* in the accountants controller, we turn `Patient.search(...)` into `Patient.search(..., limit=nil)` which I think will return all results like we intend
* we revert the changes here in dashboards_controller, which will still keep the current behavior in place

my bad for not being clearer here about what I was thinking, I apologize! Also open to an argument about this as the preferred approach - let me know what you think.

Reply:
oh, very fair, that seems safer for future search use-cases. makes sense!

Reply:
turns out passing `.limit(nil)` does nothing, so we get that functionality for free!